configgen/recalboxFiles.py

```diff
@@ -64,7 +69,27 @@
 ppssppControls = CONF + '/ppsspp/PSP/SYSTEM/controls.ini'
 ppssppControlsInit = HOME_INIT + 'configs/ppsspp/PSP/SYSTEM/controls.ini'
 
+dosboxBin = '/usr/bin/doxbox'
 dosboxCustom = CONF + '/dosbox'
 dosboxConfig = dosboxCustom + '/dosbox.conf'
 
-logdir = HOME + '/logs/'
+linappleBin = '/usr/bin/linapple'
+
+scummvmBin = '/usr/bin/scummvm'
+scummvmSaves = SAVES + '/scummvm'
+
+viceBin = '/usr/bin/x64'
+viceConfig = CONF + "/vice/vice.conf"
+
+# Please keep this one at the end, and complete with your new binaries
+recalboxBins = [dosboxBin,
```

Comment:
ok. 
Maybe we should think about a way to gather systems in configgen. Now we have to declare them in both emulatorlauncher and recalboxFiles. There must be a more clever way. Maybe add some attributes to the Emulator class ?